src/pages/Lending/Pool/components/Borrow/BorrowInput.tsx

```diff
@@ -338,6 +367,14 @@ export const BorrowInput = ({
     setConfirmedQuote(lendingQuoteData ?? null)
   }, [isLendingQuoteSuccess, lendingQuoteData, setConfirmedQuote])
 
+  useEffect(() => {
+    if (!lendingQuoteData) return
+    const _quoteSlippageDecimalPercentage = bnOrZero(lendingQuoteData.quoteSlippageBorrowedAssetUsd)
+      .div(lendingQuoteData.quoteCollateralAmountFiatUsd)
+      .toNumber()
+    setQuoteSlippageDecimalPercentage(_quoteSlippageDecimalPercentage)
```

Comment:
@0xApotheosis since this reacts on `lendingQuoteData`, feels like we could remove the state field and have a `useMemo()` for this logic instead

Reply:
That's cleaner! Addressed in https://github.com/shapeshift/web/pull/6569/commits/6b2888c1d04597f110cd4fddbed0ee38867bc2c8.